src/core/execute.c

```diff
@@ -4250,7 +4250,9 @@ static int exec_child(
                 }
 
                 r = cg_attach_everywhere(params->cgroup_supported, p, 0, NULL, NULL);
-                if (r < 0) {
+                if (r == -EUCLEAN)
+                        log_unit_warning(unit, "Couldn't attach process to cgroup %s. The threaded mode is detected, ignoring.", p);
```

Comment:
I think an explicit error message here is a good idea, but I think we still should fail here, as before with EXIT_CGROUP. The daemon is doing something wrong to place a threaded cgroup immediately below the delegated one, and we should make that discoverable easily, via a friendly log message, but not hide handle this gracefully otherwise.